GitExtUtils/FileSystemUtils.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.IO;
+
+namespace GitExtUtils
+{
+    public static class FileSystemUtils
+    {
+        public static bool IsRootedDirectoryPath(string path)
```

Comment:
I don't see any need for a new public API. Not only it serves a very specific use case, it also requires extensive testing (significantly more than targeting the specific case).
Please move this method into `FormInit` as a private method and test if via the `TestAccessor` pattern we use.